sd-finetuner-workflow/sd-finetuner/finetuner.py

```diff
@@ -526,62 +514,75 @@ def sample(self, prompt: str) -> None:
         gc.collect()
 
     def step(self, batch: dict, epoch: int) -> dict:
-        # Convert images to latent space
-        latents = self.vae.encode(
-            batch["pixel_values"].to(self.device, dtype=self.weight_dtype)
-        ).latent_dist.sample()
-        latents = latents * 0.18215
-
-        # Sample noise
-        noise = torch.randn_like(latents)
-        bsz = latents.shape[0]
-        # Sample a random timestep for each image
-        timesteps = torch.randint(
-            0,
-            self.noise_scheduler.num_train_timesteps,
-            (bsz,),
-            device=latents.device,
-        )
-        timesteps = timesteps.long()
+        train_loss = 0.0
+        with self.accelerator.accumulate(self.unet):
+            # Convert images to latent space
+            latents = self.vae.encode(
+                batch["pixel_values"].to(self.weight_dtype)
+            ).latent_dist.sample()
+            latents = latents * 0.18215
```

Comment:
What is this magic number? Should this be a constant?

Reply:
It is a constant used for scaling the latents https://github.com/huggingface/diffusers/issues/437, the original purpose being to handle different latent spaces from different autoencoders.